.github/workflows/python.yml

```
@@ -0,0 +1,229 @@
+    branches: [main]
+  release:
+    types: [published]
+
+name: Python
+
+jobs:
+  macos:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v5
+      - uses: actions/setup-python@v4
```

actionlint (1.7.8)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)